modules/file-system/pre-existing-network-storage/templates/ddn_exascaler_luster_client_install.tftpl

```diff
@@ -46,3 +46,5 @@ do
 done
 chmod +x /usr/sbin/esc-client
 esc-client auto setup --config /etc/esc-client.conf
+esc-client unmount -m "${local_mount}"
+mv /etc/fstab.last /etc/fstab
```

Comment:
what are these lines here? why do you need to remove it from the fstab? (won't the mount be a no-op in that case?)